### PR TITLE
Enum support for AuthenticatedState in ts

### PR DESCRIPTION
**Pull request by cacheung**
Enum support for AuthenticatedState in ts.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/edgeidentity/__tests__/IdentityMapTests.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ describe('IdentityMap', () => {
 
     let itemsForNamespace2 = idMap.getIdentityItemsForNamespace(namespace2);
     expect(itemsForNamespace2.length).toEqual(2);
-    expect(itemsForNamespace2).toEqual([{"id": identifier3, "authenticatedState": authenticatedState3, "primary": false}, {"id": identifier4, "authenticatedState": AuthenticatedState.AMBIGUOUS, "primary": false}]);
+    expect(itemsForNamespace2).toEqual([{"id": identifier3, "authenticatedState": 'loggedOut', "primary": false}, {"id": identifier4, "authenticatedState": AuthenticatedState.AMBIGUOUS, "primary": false}]);
```
Comment (emdobrin) [inline packages/edgeidentity/__tests__/IdentityMapTests.ts:56]: Was this change required for identifier3?

**Review by emdobrin — APPROVED** [review]

File: packages/edgeidentity/__tests__/IdentityMapTests.ts (hunk 1 above)
Comment (cacheung) [reply packages/edgeidentity/__tests__/IdentityMapTests.ts:56]: I want to check the test shows 'loggedOut'.
Since AuthenticatedState.LOGGED_OUT  case is similar to AuthenticatedState.AMBIGUOUS which is checked in the identifier4.
